Documentation/getting-started.md

```diff
@@ -24,12 +24,27 @@ On first build Torus will use [glide](https://github.com/Masterminds/glide) to d
 ### 1) Get etcd
 You need a *v3.0* or higher [etcd](https://github.com/coreos/etcd) instance, as torus uses the v3 API natively and uses the latest client. You might try [etcd v3.0.0-beta.0](https://github.com/coreos/etcd/releases/tag/v3.0.0-beta.0). 
 
+#### Running etcd manually
 To run a single node cluster locally:
 
 ```
 etcd --data-dir /tmp/etcd
 ```
 
+#### Running etcd using rkt
```

Comment:
@s-urbaniak Let's move this section to https://github.com/coreos/etcd/blob/master/Documentation/op-guide/container.md#run-etcd-clusters-inside-containers and simply link this to etcd doc? Then it would be easier for us to maintain the doc.